src/Microsoft.ML.OnnxTransform/DnnImageFeaturizerTransform.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Microsoft.ML.StaticPipe;
+using Microsoft.ML.StaticPipe.Runtime;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.ML.Transforms
+{
+    // This is a helper class that is required to use the DnnImageFeaturizer estimator.
+    // Note that by default, it is not usable as it does not have any valid methods that return an EstimatorChain that
```

Comment:
I feel we still should do reference. You refer some types because they are used either explicitly or implicitly.

---
In reply to: [232397881](https://github.com/dotnet/machinelearning/pull/1447#discussion_r232397881) [](ancestors = 232397881,232382991)

Reply:
Thanks for the suggestion. I added refs to everything that's not the model project themselves.

---
In reply to: [232413934](https://github.com/dotnet/machinelearning/pull/1447#discussion_r232413934) [](ancestors = 232413934,232397881,232382991)